index.html

```diff
@@ -28,8 +28,8 @@
       </nav>
     </header>
     <main>
-      <div class="binabox">
-      <h1 class="binabox">
+      <div class="binabox__title">
```

Comment:
just "binabox" here - it's a block. h1 class="binabox_title" is an element of the block